docs/tests/integration/test_logcollector/test_location/index.md

```diff
@@ -0,0 +1,22 @@
+# Test location
+
+## Overview 
+
+These tests check if the introduced configuration works as expected for valid values and if the logcollector 
+response for analyzing requests works properly.
```

Comment:
```suggestion
Check if the specified location values work as expected. 
```

Reply:
Fixed in 8e8e84a2a739ed54d49446ac6e5db42ccb2d59dd